contrib/libxslt/sapi_libxslt_test.cc

```diff
@@ -0,0 +1,121 @@
+/*
+ * libxslt_tutorial.c: demo program for the XSL Transformation 1.0 engine
+ *
+ * based on xsltproc.c, by Daniel.Veillard@imag.fr
+ * by John Fleck
+ *
+ * See Copyright for the status of this software.
+ *
+ */
```

Comment:
Please replace with our standard header (same in sample_clsproc.cc)

Reply:
This code is derived from the XSLTproc source code.  That code is under an MIT license so there should be no problem using it, but the copyright notice and license text will need to be included somewhere.

Reply:
In an internal discussion we thought it's better to use our header in this instance.

Reply:
I think I should've been clearer, I actually meant having both, especially the original author. The changes to the file will be under Google copyright.